docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -159,7 +159,7 @@ $ docker run -d  --rm \
     -p 19998:19998 \
     --net=alluxio_network \
     --name=alluxio-master \
-    -e ALLUXIO_JAVA_OPTS="-Dalluxio.master.hostname=alluxio-master -Dalluxio.master.mount.table.root.ufs=/opt/alluxio/underFSStorage" \
+    -e ALLUXIO_JAVA_OPTS="-Dalluxio.master.hostname=alluxio-master -Dalluxio.master.mount.table.root.ufs=/opt/alluxio/underFSStorage -Dalluxio.worker.memory.size=1G" \
```

Comment:
This is the master container. Why are we adding this to the master? Setting the worker memory size on the master is not required

Reply:
Actually it is necessary because Alluxio Workers'  ram capacity will be 2/3 of total system memory  although the conf of ram capacity  in Alluxio Workers' side is 1GB checked by "./bin/alluxio getConf"

Reply:
i see you're saying that in the following line in `alluxo-mount.sh` in the worker containter will RPC the master to get the worker conf?
```
bin/alluxio-mount.sh:  MEM_SIZE=$(${BIN}/alluxio getConf --unit B alluxio.worker.memory.size)
```

if this is the case, then we need to fix `alluxio-mount.sh` to look at local conf instead of adding worker properties to the master

Reply:
start master & worker without conf in master 
![image](https://user-images.githubusercontent.com/22467785/78968062-60c20580-7b36-11ea-8bfd-e64b392f4571.png)

Reply:
worker's ram capacity is 2/3 to total ram
![image](https://user-images.githubusercontent.com/22467785/78968254-a383dd80-7b36-11ea-97b5-696474888dec.png)
![image](https://user-images.githubusercontent.com/22467785/78968313-bd252500-7b36-11ea-84dd-385cfdc34f4c.png)

Reply:
worker's conf is right but doesn't worked
![image](https://user-images.githubusercontent.com/22467785/78968389-07a6a180-7b37-11ea-929c-1da804eeeed3.png)

Reply:
redeploy with the conf in master
![image](https://user-images.githubusercontent.com/22467785/78968527-71bf4680-7b37-11ea-98c6-7f526ae9ebee.png)

Reply:
working as except
![image](https://user-images.githubusercontent.com/22467785/78968656-c236a400-7b37-11ea-88a5-9d65ff17e0f7.png)
![image](https://user-images.githubusercontent.com/22467785/78968689-d084c000-7b37-11ea-8be6-4eb3cb80c098.png)

Reply:
working as except
![image](https://user-images.githubusercontent.com/22467785/78971714-c1edd700-7b3e-11ea-97a0-5dfd9e453cd4.png)
![image](https://user-images.githubusercontent.com/22467785/78971739-cd410280-7b3e-11ea-8339-de0cb6d1df76.png)